apps/etterlatte-saksbehandling-ui/client/src/components/aktivitetsplikt/OppgaveVurderingRoute.tsx

```diff
@@ -0,0 +1,49 @@
+import { OppgaveDTO } from '~shared/types/oppgave'
+import { StatusBar } from '~shared/statusbar/Statusbar'
+import { GridContainer, MainContent } from '~shared/styled'
+import { Navigate, Route, Routes } from 'react-router-dom'
+import React, { createContext, useContext } from 'react'
+import { AktivitetspliktSidemeny } from '~components/aktivitetsplikt/sidemeny/AktivitetspliktSidemeny'
+import {
+  AktivitetspliktSteg,
+  AktivitetspliktStegmeny,
+} from '~components/aktivitetsplikt/stegmeny/AktivitetspliktStegmeny'
+import { VurderAktivitet } from '~components/aktivitetsplikt/vurdering/VurderAktivitet'
+import { VurderingInfoBrev } from '~components/aktivitetsplikt/brev/VurderingInfoBrev'
+
+const AktivitetspliktOppgaveContext = createContext<OppgaveDTO>({} as OppgaveDTO)
```

Comment:
Kanskje litt hyper optimalisering med kontekst her nå men :P

Reply:
Intensjonen var mer at man da kan hente ut oppgaven vi gjør vurderingen for, uten å trenge å drille prop ned

Reply:
nice